modules/serverless-rn-1-14-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-14-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.14.0
+
+[id="new-features-1-14-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.20.0.
+* {ServerlessProductName} uses Knative Eventing 0.20.0.
+* {ServerlessProductName} now uses Kourier 0.20.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.20.0.
+* {ServerlessProductName} now uses Knative Kafka 0.20.0.
+* Apache Kafka on {ServerlessProductName} is now Generally Available (GA).
++
+[IMPORTANT]
+====
+Only the `v1beta1` version of the API for Apache Kafka on {ServerlessProductName} is supported. Do not use the `v1alpha1` version of the API, as this is deprecated.
+====
+* The Operator channel for installing and upgrading {ServerlessProductName} has been updated to `stable` for {product-title} 4.6 and newer versions.
+* {ServerlessProductName} is now supported on IBM Power Systems, IBM Z, and LinuxONE, except for the following features, which are not yet supported:
+** Apache Kafka functionality.
+** {ServerlessProductName} Functions developer preview.
+// Not including Camel-K since we don't document or support that yet for serverless anyway.
+
+[id="fixed-issues-1-14-0_{context}"]
+== Fixed issues
+
+// confirm with QE
+
+[id="known-issues-1-14-0_{context}"]
+== Known issues
+
+* Subscriptions for the Kafka channel sometimes fail to become marked as `READY` and remain in the `SubscriptionNotMarkedReadyByChannel` state. You can fix this by restarting the controller for the Kafka channel.
```

Comment:
May be removed if bug is fixed, needs to be confirmed by @matzew or whoever is reviewing for QE

Reply:
Please see @maschmid 's comment here: https://issues.redhat.com/browse/SRVKE-779?focusedCommentId=16092984&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16092984

Reply:
yeah, agreed - I'd leave it b/c of 779